channels/1ds-post-js/src/HttpManager.ts

```diff
@@ -470,6 +470,23 @@ export class HttpManager {
                     requestInit.headers = payload.headers;
                 }
 
+                const handleResponse = (status, headerMap, responseText) => {
```

Comment:
Do these need type qualifiers?
ie. Is TypeScript happy with these or is just assuming that they are `any`, if it thinks they are `any` then we (should) try and give them specific types. Although the `headerMap` is an object (`any`) always anyway.

Reply:
If TypeScript (if VSCode) inferes the type's then don't worry about defining them.